editor.planx.uk/src/@planx/components/PlanningConstraints/Public.tsx

```diff
@@ -125,12 +126,12 @@ function Component(props: Props) {
   );
 
   // XXX handle both/either Digital Land response and custom GIS hookup responses; merge roads for a unified list of constraints
-  const constraints: GISResponse["constraints"] | {} = {
+  const constraints: GISResponse["constraints"] | Record<string, any> = {
```

Comment:
There's a few types that are a bit messy here. 

GIS types are mostly managed in planx-core, and I'll plan to open a followup PR to try to tidy some of this - we're notably missing a Planning Data "entity" type which we've avoided in the past because of non-standard fields per dataset - but I think we should be able to get a decent union type in place to eliminate some of these newly introduced `any`s